source/api/content-state/0.9/index.md

```diff
@@ -0,0 +1,713 @@
+
+A content state annotation can take several formats. All of the four examples below are equivalent to the following annotation, which is the simplest content state: a reference to a IIIF Manifest.
+
+```json
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "https://example.org/Annotation-server/bookmarks/b1",
+  "type": "Annotation",
+  "motivation": ["contentState"],
+  "target": {
+     "id": "https://example.org/iiif/item1/manifest",
+     "type": "Manifest"
+  }
+}
+```
+
+Publishers _SHOULD_ provide the content state annotation in one of the following forms.
+A client _SHOULD_ be able to accept and process the content state in all of these forms. 
+
+#### 2.2.1 Full Annotation
+
+The content state _MAY_ be supplied as JSON-LD, as a full Annotation with the motivation `contentState`, as in the example above.
+
+The target of the annotation is in this case a complete IIIF resource (here, a Manifest) but in more complex cases, the target could be a part of a IIIF resource.  
+
+#### 2.2.2 Annotation URI
+
+The content state _MAY_ be supplied as a string whose value is the URI of an Annotation with the motivation `contentState`, that the client must de-reference and process. For the example above, this would be the URI `https://example.org/Annotation-server/bookmarks/b1`. The response from that URI would be the JSON above.
+
+#### 2.2.3 Target body
+
+The content state _MAY_ be supplied as JSON-LD, as the value of the `target` property of an implied Annotation with the motivation `contentState`. For the example above, this would be:
+
+```json
+{
+    "id": "https://example.org/iiif/item1/manifest",
+    "type": "Manifest"
+}
+```
+
+This form is better suited to scenarios where compactness is important, for example a query string parameter, where it is will be combined with base64url encoding as described in Section 2.3.
+
+#### 2.2.4 Target URI
+
+ The content state _MAY_ be supplied as a string whose value is the `id` (the dereferenceable URI) of the `target` property only. This is the simplest form and is just the URI of a resource. For the example above, this would be the URI `https://example.org/iiif/item1/manifest`. The client would simply load this Manifest and display it. 
+
+Examples 2.2.2 and 2.2.4 are both URIs. It is up to the client to recognise that 2.2.4 is a Manifest, whereas 2.2.2 is an Annotation that points to a Manifest. The client inspects the `type` property to determine what the de-referenced resource is.
```

Comment:
Suggest adding that if the type is Annotation, then the client must also look at `motivation` to determine if it is a contentState or not.  I think this should also have a normative MUST, otherwise it won't work.

Reply:
In this scenario, where you have found a resource that you expect to be a content state (because it's the value of a `iiif-content` attribute or whatever), if the resource is an annotation and the `motivation` is **not** contentState, then the content being addressed is the annotation itself, rather than the target of the annotation.

This is valid (the target is implied, and can be any IIIF resource), but probably very rare. "I want you to look at this tagging annotation". I've now tried to explain it here.